auth/PhabricatorBMOAuth.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @provides bmo-auth-js
+ */
+
+// Display the admin login form if "?admin"
+if((new URLSearchParams(window.location.search)).has('admin')) {
+  var adminForm = document.querySelector('form["action=/auth/login/password:self/"]');
```

Comment:
first quote in wrong place:

var adminForm = document.querySelector('form[action="/auth/login/password:self/"]');